src/SettingsPage/View.php

```diff
@@ -59,6 +62,35 @@ public function add_options_page() {
 	 */
 	public function render_page() {
 
+		$this->register_scripts_styles();
+
+		$tabs = [
+			new AdminBar(),
+			new Dashboard(),
+			new Menu()
+		];
+
 		include $this->settings_page->get_template_path() . '/Templates/SettingsPage.php';
 	}
-}
\ No newline at end of file
+
+
+	/**
+	 * Load the admin scripts and styles
+	 *
+	 * @return    bool
+	 */
+	private function register_scripts_styles(): bool {
+
+		wp_enqueue_script(
+			'adminimize-admin-scripts',
+			plugins_url('../../assets/js/adminimize.js', __FILE__),
```

Comment:
`jquery-ui-tabs` is a part of the core, there we should use, see https://developer.wordpress.org/reference/functions/wp_enqueue_script/#default-scripts-included-and-registered-by-wordpress A custom implementation is not, should not necessary.

Reply:
I have included `jquery-ui-tabs` as a dependency so the adminimize script is loaded after the core loads tabs script. If I do not include that then there is a possibility of JS error depending on the order in which the scripts are included.